packages/cli/package.json

```diff
@@ -77,30 +77,30 @@
     "all": true
   },
   "dependencies": {
-    "@igniteui/angular-templates": "~9.1.520",
```

Comment:
Do not change from ~ to ^ for igniteui-angular, because we want to stick to patch updates